xsd/NeTEx_publication_timetable.xsd

```diff
@@ -1285,17 +1285,6 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====Organisation============================== -->
```

Comment:
Investigate if this deletion actually is correct. I remember undoing it.

Reply:
it seems to me that it is is still in the file on line 1376ff